app/views/users/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<section class="my-bio bg-secondary bg-gradient">
+  <div class="container">
+    <div class="row">
+      <%= render partial: 'user', locals: { user: @user } %>
+    </div>
+    <div class="card my-4">
+      <%= render partial: 'bio', locals: { user: @user } %>
+    </div>
+    <%= link_to "Add a new post", new_user_post_path(user_id: @user.id), class: "btn btn-primary"%>
```

Comment:
- [ ] Please fix this error that occurs when I click a user as per the requirement.
![image](https://github.com/Izebiafe/Blog/assets/91486791/17000d90-d086-4ac5-a3d4-7ebca298b8e5)